cpp/CMakeLists.txt

```diff
@@ -744,3 +745,81 @@ add_custom_command(OUTPUT CUDF_DOXYGEN
                    VERBATIM)
 
 add_custom_target(docs_cudf DEPENDS CUDF_DOXYGEN)
+
+
+####################################################################################################
+# - cudf_kafka - OFF by default due to increased number of dependencies
+if(BUILD_CUDF_KAFKA)
+
+    # cudf_kafka library
+    add_library(libcudf_kafka
+        libcudf_kafka/src/kafka_consumer.cpp
+    )
+
+    # Include paths
+    include_directories("${CMAKE_SOURCE_DIR}/libcudf_kafka/include"
+                        "${CMAKE_CURRENT_SOURCE_DIR}/include/cudf")
+    
+    # Rename installation to proper name for later finding
+    set_target_properties(libcudf_kafka PROPERTIES OUTPUT_NAME "cudf_kafka")
+    set_target_properties(libcudf_kafka PROPERTIES BUILD_RPATH "\$ORIGIN")
+
+    ###################################################################################################
+    # cudf_kafka - librdkafka -------------------------------------------------------------------------
+
+    find_path(RDKAFKA_INCLUDE "librdkafka" HINTS "$ENV{RDKAFKA_ROOT}/include")
+    find_library(RDKAFKA++_LIBRARY "rdkafka++" HINTS "$ENV{RDKAFKA_ROOT}/lib" "$ENV{RDKAFKA_ROOT}/build")
+
+    message(STATUS "RDKAFKA: RDKAFKA++_LIBRARY set to ${RDKAFKA++_LIBRARY}")
+    message(STATUS "RDKAFKA: RDKAFKA_INCLUDE set to ${RDKAFKA_INCLUDE}")
+
+    target_link_libraries(libcudf_kafka ${RDKAFKA++_LIBRARY})
+    include_directories("${RDKAFKA_INCLUDE}")
+
+    ###################################################################################################
+    # - cudf_kafka Install ----------------------------------------------------------------------------
+    target_link_libraries(libcudf_kafka cudf)
+
+    install(TARGETS libcudf_kafka
+            DESTINATION libcudf_kafka/lib
+            COMPONENT libcudf_kafka)
+
+    install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/libcudf_kafka/include
+            DESTINATION libcudf_kafka/include
+            COMPONENT libcudf_kafka)
```

Comment:
This is still incorrect. We should install into `include`, but that install should land in a `cudf_kafka` folder for example. I'd suggest making the directory structure `cpp/libcudf_kafka/include/cudf_kafka/kafka_consumer.hpp`. And then this command can be updated to:
```
    install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/libcudf_kafka/include
            DESTINATION include
            COMPONENT libcudf_kafka)
```